components/CountryTagsInput.js

```diff
@@ -0,0 +1,37 @@
+// Module imports
+import TagsInput from './TagsInput'
+
+
+
+
+
+// Component imports
+import countryList from '../data/CountryList'
+
+
+
+
+
+export default class extends TagsInput {
+  search (_query) {
+    const query = _query.toLowerCase()
+    if (query) {
+      const response = countryList.filter(country => country.toLowerCase().startsWith(query.toLowerCase()))
+
+      if (query.startsWith('us')) {
```

Comment:
What are these exceptions all about?

Reply:
for people to attempt to type in "US[A]" for "united states". The entire TagsInput component is due for a rewrite, and this won't be needed in the future. I'm doing a much smarter search for the replacement.